lib/components/home/artwork_rails/artwork_rail_header.js

```diff
@@ -3,25 +3,51 @@
 
 import Relay from 'react-relay'
 import React from 'react'
-import { StyleSheet, Dimensions, View, Text } from 'react-native'
+import { StyleSheet, Dimensions, View, Text, TouchableWithoutFeedback } from 'react-native'
 
 import SerifText from '../../text/serif'
 import colors from '../../../../data/colors'
 import Button from '../../buttons/inverted_button'
+import SwitchBoard from '../../../modules/switch_board'
+import fragments from './relay_fragments'
```

Comment:
Maybe I'll move `Switchboard` into a file with the fragments and just have methods for routing and relay. Like a network model.